tests/cypress/config/ImageManifest_governance/policy-config.yaml

```diff
@@ -1,6 +1,6 @@
 # these are policies required by following Polarion test cases:
 
-test-imagemanifest-policy-[ID]:
+t-imp-[ID]:
```

Comment:
why is it `t-imp`?

Reply:
I needed to shorten it to make the fuzzy search more reliable, `t-imp` = **T**est **I**mage**M**anifest**P**olicy, because I didn't want to completely lose the ability to search for where this policy came from in the code.

Reply:
I see. Can you add this as a comment to this line? It will help explain.